app/frontend/components/pages/login/hardwareAuth.tsx

```diff
@@ -80,40 +86,42 @@ const LoadByHardwareWalletSection = () => {
           <div className="authentication-paragraph small">
             {LedgerAffiliateLink('Support us by buying one')}
           </div>
-          <button
-            {...tooltip(
-              'Support for Ledger is temporarily disabled',
-              !ADALITE_CONFIG.ADALITE_ENABLE_LEDGER
-            )}
-            disabled={!ADALITE_CONFIG.ADALITE_ENABLE_LEDGER}
-            className="button primary ledger thin-data-balloon"
-            onClick={() =>
-              loadWallet({
-                cryptoProviderType: CryptoProviderType.LEDGER,
-                shouldExportPubKeyBulk: enableBulkExport,
-              })
-            }
-          >
-            Unlock with
-            <div className="ledger-logo-container">
-              <LedgerLogoWhite />
+          <div>
+            <button
+              {...tooltip(
+                'Support for Ledger is temporarily disabled',
+                !ADALITE_CONFIG.ADALITE_ENABLE_LEDGER
+              )}
+              disabled={!ADALITE_CONFIG.ADALITE_ENABLE_LEDGER}
+              className={`button primary ledger thin-data-balloon ${styles.ledgerButton}`}
+              onClick={() =>
+                loadWallet({
+                  cryptoProviderType: CryptoProviderType.LEDGER,
+                  shouldExportPubKeyBulk: enableBulkExport,
+                  selectedLedgerTransportType,
+                })
+              }
+            >
+              Unlock with
+              <div className="ledger-logo-container">
+                <LedgerLogoWhite />
+              </div>
+            </button>
+            <div className={styles.transportWrapper}>
+              <div className={styles.transportLabel}>
+                Transport
```

Comment:
Regular users won't have a clue what "transport" is, I'd put in the UI "Method" instead

Reply:
Fixed